tesseract/tesseract_kinematics/include/tesseract_kinematics/core/validate.h

```diff
@@ -100,19 +145,32 @@ inline bool checkKinematics(const tesseract_kinematics::ForwardKinematics::Const
   seed_angles.setZero();
   joint_angles2.setZero();
 
-  for (int t = 0; t < fwd_kin->numJoints(); ++t)
+  int nj = static_cast<int>(fwd_kin->numJoints());
```

Comment:
const int?

Reply:
I know clang complains about this in the function arguments but not sure about in the body. I will change it and see.

Reply:
It looks like clang only cares when it is one of the function arguments.